### ISSUE TITLE
ValueError: Unknown file extension: .xlsx.

### ISSUE THREAD

**Issue by J535D165** [msdslab/automated-systematic-review#142]
```
Jonathans-MacBook-Pro:ASReview_tryout jonathandebruin$ asreview oracle Nagtegaal\ Data.xlsx 

            _____ _____            _
     /\    / ____|  __ \          (_)
    /  \  | (___ | |__) |_____   ___  _____      __
   / /\ \  \___ \|  _  // _ \ \ / / |/ _ \ \ /\ / /
  / ____ \ ____) | | \ \  __/\ V /| |  __/\ V  V /
 /_/    \_\_____/|_|  \_\___| \_/ |_|\___| \_/\_/

---------------------------------------------------------------------------------
|                                                                                |
|  Welcome to the ASReview Automated Systematic Review software.                 |
|  In this mode the computer will assist you in creating your systematic review. |
|  After giving it a few papers that are either included or excluded,            |
|  it will compute a model and show progressively more relevant papers.          |
|                                                                                |
|  GitHub page:        https://github.com/msdslab/automated-systematic-review    |
|  Questions/remarks:  asreview@uu.nl                                            |
|                                                                                |
---------------------------------------------------------------------------------

? Please provide a file to store the results of your review:  test.csv
Traceback (most recent call last):
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/bin/asreview", line 10, in <module>
    sys.exit(main())
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/__main__.py", line 250, in main
    _review_general(sys.argv[1])
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/__main__.py", line 235, in _review_general
    review_oracle(path, **args_dict)
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/review/factory.py", line 278, in review_oracle
    review(dataset, *args, mode='oracle', log_file=log_file, **kwargs)
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/review/factory.py", line 200, in review
    reviewer = get_reviewer(*args, mode=mode, model=model, **kwargs)
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/review/factory.py", line 109, in get_reviewer
    abstract_only=settings.abstract_only)
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/readers.py", line 151, in from_file
    return cls.from_data_frame(_df_from_file(fp), *args, **kwargs)
  File "/Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages/asreview/readers.py", line 349, in _df_from_file
    raise ValueError(f"Unknown file extension: {Path(fp).suffix}.\n"
ValueError: Unknown file extension: .xlsx.
from file Nagtegaal Data.xlsx
Jonathans-MacBook-Pro:ASReview_tryout jonathandebruin$ pip show asreview
Name: asreview
Version: 0.3.1
Summary: Automated Systematic Review
Home-page: https://github.com/msdslab/automated-systematic-review
Author: ASReview Core Development Team
Author-email: asreview@uu.nl
License: UNKNOWN
Location: /Users/jonathandebruin/.pyenv/versions/3.7.4/lib/python3.7/site-packages
Requires: RISparser, dill, tensorflow, modAL, sklearn, fuzzywuzzy, h5py, numpy, PyInquirer, pandas, xlrd
Required-by: 
Jonathans-MacBook-Pro:ASReview_tryout jonathandebruin$ 
```

**Comment by J535D165** [comment]
Fix in 0.3.2.

**Comment by J535D165** [comment]
Closed in #143
